### DISCUSSION TITLE
JLCPCB order

### DISCUSSION THREAD

**Discussion by Ulf349** [revk/xESP32-Faikout#427] [General]
Hi, I would like to order the Faikin module at JLCPCB. In the description is the size with 70x70mm. If I upload the Gerber and BOM data to JLCPCB the ESP can only be assembled with Standard PCB instead of Economy and the size will be increase from 70x70mm to 80x70mm automatically. Can somebody help me?

**Comment by revk** [comment]
They won't do Economy with an ESP32 on the board, so you need Standard anyway.

**Reply by Ulf349** [reply]
Thanks for your answer. Is it better to order directly Faikin+ or the normal Faikin? Which do you prefer?

**Reply by revk** [reply]
Faikin+ is a double sided test board, you want Faikin

You need a TC2030 lead to flash it
